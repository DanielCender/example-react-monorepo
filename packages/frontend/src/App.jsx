import React from 'react';

const App = () => {
  return <div>This is a sample App Component.</div>;
};

export default App;
