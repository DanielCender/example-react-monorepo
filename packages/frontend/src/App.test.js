import React from 'react';
import App from './App';

test('Should render correctly', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
