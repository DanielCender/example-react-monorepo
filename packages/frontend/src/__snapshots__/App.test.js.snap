// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  This is a sample App Component.
</div>
`;
